makefile

```diff
@@ -9,13 +9,19 @@ dev: check-dep
 	@echo "$(P) Run \`npm run dev\` of all packages with \`scripts.dev\`"
 	$(BIN_DIR)/lerna run --parallel dev
 
-build: check-dep
+build: clean build-lib build-dist 
+
+build-lib: check-dep
 	@echo "$(P) Run \`npm run build\` of all packages with \`scripts.build\`"
 	$(BIN_DIR)/lerna run --stream build
+	NODE_ENV=production $(BIN_DIR)/babel packages/index.js --out-dir lib --root-mode upward
```

Comment:
We will have a `index.js` file (not included in this PR) like this:

```
    // index.js

    import ScrollableImage from './scrollable-image/src'

    export { ScrollableImage }
```

The reason to add this file is we need to support named imports, like:

```
   import { ScrollableImage } from '@twreporter/orangutan'
```